app/uk/gov/hmrc/cgtpropertydisposals/service/returns/ReturnsService.scala

```diff
@@ -102,9 +104,77 @@ class DefaultReturnsService @Inject() (
                               logger.warn("Could not send return submission confirmation email or audit event", e)
                               EitherT.pure[Future, Error](())
                             }
+      _                  <- handleAmendedReturn(returnRequest)
     } yield returnResponse
   }
 
+  private def handleAmendedReturn(submitReturnRequest: SubmitReturnRequest): EitherT[Future, Error, Unit] = {
+    def modifyDraftReturn(draftReturn: DraftReturn): Option[DraftReturn] =
+      if (
+        draftReturn.exemptionAndLossesAnswers.isEmpty && draftReturn.yearToDateLiabilityAnswers.isEmpty && draftReturn.supportingEvidenceAnswers.isEmpty
+      )
+        None
+      else
+        draftReturn match {
+          case single: DraftSingleDisposalReturn                      =>
+            Some(
+              single.copy(
+                exemptionAndLossesAnswers = None,
+                yearToDateLiabilityAnswers = None,
+                supportingEvidenceAnswers = None
+              )
+            )
+          case multiple: DraftMultipleDisposalsReturn                 =>
+            Some(
+              multiple.copy(
+                exemptionAndLossesAnswers = None,
+                yearToDateLiabilityAnswers = None,
+                supportingEvidenceAnswers = None
+              )
+            )
+          case singleIndirect: DraftSingleIndirectDisposalReturn      =>
+            Some(
+              singleIndirect.copy(
+                exemptionAndLossesAnswers = None,
+                yearToDateLiabilityAnswers = None,
+                supportingEvidenceAnswers = None
+              )
+            )
+          case multipleIndirect: DraftMultipleIndirectDisposalsReturn =>
+            Some(
+              multipleIndirect.copy(
+                exemptionAndLossesAnswers = None,
+                yearToDateLiabilityAnswers = None,
+                supportingEvidenceAnswers = None
+              )
+            )
+          case singleMixedUse: DraftSingleMixedUseDisposalReturn      =>
+            Some(
+              singleMixedUse.copy(
+                exemptionAndLossesAnswers = None,
+                yearToDateLiabilityAnswers = None,
+                supportingEvidenceAnswers = None
+              )
+            )
+        }
+
+    val cgtReference = submitReturnRequest.subscribedDetails.cgtReference
+    if (submitReturnRequest.originalReturnFormBundleId.isEmpty)
+      EitherT.pure(())
+    else {
+      import cats.instances.list._
```

Comment:
does this cause conflicts if places at the top of the file with the other imports?

Reply:
yeah, in some other places it gets confused with the future cats instances in other for-comprehensions